18F/identity-idp
# LG-9869: Allow users to show information via reauthentication in account profile

## 🎫 Ticket

[LG-9869](https://cm-jira.usa.gov/browse/LG-9869)

## Testing Plan
1. Have a user fully verified
2. go to account page
3. Wait until locked session so pii is hidden (30 or so minutes)
4. click on authenticate to view your information
5. go through submitting password and MFA
6. Ensure you are able to see verified information again. 

## 👀 Screenshots
This is a video of the new functionality working as expected with users seeing their information after authenticating. 
(https://github.com/18F/identity-idp/assets/23225522/a9d83387-e33a-433f-a8d3-1430e08caec2)
